### PR TITLE
Added sitemap plugin to gatsby-config

### PR DESCRIPTION
**Pull request by ozairashfaqueSSW**
#410 

- Added [gatsby-plugin-sitemap](https://www.gatsbyjs.com/plugins/gatsby-plugin-sitemap/) to gatsby-config

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ const getPlugins = () => {
       },
     },
     'gatsby-plugin-eslint',
+    'gatsby-plugin-sitemap',
```
Comment (bradystroud) [inline gatsby-config.js:35]: Might need some configuration

> You probably do not want to use the defaults in this plugin

See https://www.gatsbyjs.com/plugins/gatsby-plugin-sitemap/#recommended-usage

File: package.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "babel-plugin-styled-components": "^1.11.1",
     "classnames": "^2.2.6",
     "crypto": "^1.0.1",
-    "gatsby": "^2.24.67",
+    "gatsby": ">=1",
```
Comment (bradystroud) [inline package.json:39]: What does this do?
I think it would be better to lock it on a certain version as there could be breaking changes

**Review by bradystroud — CHANGES_REQUESTED** [review]

**Comment by bradystroud** [comment]
@ozairashfaqueSSW Can you give an update on this one?

**Comment by ozairashfaqueSSW** [comment]
Closing this PR because work has already been done. see issue https://github.com/SSWConsulting/SSW.People/pull/161

**Pull request closed without merging**